src/dmd/mars.d

```diff
@@ -1640,6 +1638,22 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                 params.showColumns = true;
             else if (arg == "-vgc") // https://dlang.org/dmd.html#switch-vgc
                 params.vgc = true;
+            else if (p[1] == 'v') // https://dlang.org/dmd.html#switch-v
+            {
+                if (p[2] == '\0')
+                    params.verbose = stdout;
+                else
+                {
+                    auto filename = p + 2 + (p[2] == '=');
+                    params.verbose = fopen(filename, "w");
+                    if (!params.verbose)
+                    {
+                        import core.stdc.errno : errno;
+                        error("failed to open verbose output file '%s' (errno %d)", filename, errno);
```

Comment:
You know that there's a libc function that returns an error message for the given errno.